fix: redesign 'Latest News' section in Community Newsroom

PankajKumardev opened the pull request:
**Description**
- Redesigned the "Latest News" section in the Community Newsroom for improved UI/UX on desktop screens.
- Left-aligned the section title and Twitter follow button for better readability.
- Implemented a card-based layout for articles, including publication date and headline.
- Improved responsiveness across all screen sizes.
- Minor refactor of internal content style management.

**Before**
![image](https://github.com/user-attachments/assets/d6ad1688-8d83-46aa-a1c9-26fcd0a1bdeb)

**After** 
![image](https://github.com/user-attachments/assets/4c1cacfd-57b0-4ccb-963c-d5b4047db332)


**Related issue(s)**
Fixes #3587 


**Notes**
- Short descriptions for articles are not included as they aren't in the articles.json.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Style**
  - Enhanced the visual layout of the Newsroom section for a cleaner, more organized presentation.
  - Improved spacing and alignment of the Twitter link and icon for better readability and user experience.
  - Adjusted the height constraints of the article container for a more streamlined appearance.
  - Modified the width of the AnnouncementHero component for better responsiveness on smaller screens.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

PankajKumardev committed:
improve UI responsiveness for 'Latest News' section in Community Newsroom

PankajKumardev committed:
fix the preetier part

PankajKumardev committed:
refactor: standardize quotation marks in Newsroom component

PankajKumardev committed:
liniting fix

PankajKumardev committed:
lint fix

PankajKumardev committed:
lint fix

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
fix: correct indentation in Newsroom component

PankajKumardev committed:
Merge branch 'master' of github.com:PankajKumardev/website

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev commented:
/rtm

PankajKumardev committed:
Remove duplicate tool rendering in ToolsList using Set for unique titles.

PankajKumardev committed:
Remove duplicate tool rendering in ToolsList using Set for unique titles.

PankajKumardev committed:
Merge branch 'master' of github.com:PankajKumardev/website

PankajKumardev committed:
Refactor ToolsList component to simplify rendering and remove duplicate tool handling

Siddhant-Pragyan-Sinha commented:
Hi @PankajKumardev . Talking as a fellow contributer, I feel that you are new here. 
Please do not write the /r . t . m command just like that. It is for maintainers and codeowners to do when they approve your PR and decide to merge it. 
Take a look around the other PRs and the CoC to gain a better understanding of how things work here.

PankajKumardev commented:
> Hi @PankajKumardev . Talking as a fellow contributer, I feel that you are new here. Please do not write the /r . t . m command just like that. It is for maintainers and codeowners to do when they approve your PR and decide to merge it. Take a look around the other PRs and the CoC to gain a better understanding of how things work here.

sorry , for this , i won't do it again.

Siddhant-Pragyan-Sinha commented:
> > Hi @PankajKumardev . Talking as a fellow contributer, I feel that you are new here. Please do not write the /r . t . m command just like that. It is for maintainers and codeowners to do when they approve your PR and decide to merge it. Take a look around the other PRs and the CoC to gain a better understanding of how things work here.
> 
> sorry , for this , i won't do it again.


No worries. And sorry from my side if it felt a little too serious.

PankajKumardev committed:
Merge branch 'master' into master

vishvamsinh28 commented:
@PankajKumardev LGTM

PankajKumardev commented:
> @PankajKumardev LGTM

Than, this PR is going to be merge?

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

sambhavgupta0705 commented on components/newsroom/Newsroom.tsx:
```diff
@@ -61,17 +61,17 @@ export default function Newsroom() {
           </Paragraph>
           <div className='my-5' data-testid='Newsroom-Twitter-Link'>
-            <TextLink href='https://twitter.com/AsyncAPISpec' className='mt-4' target='_blank'>
+            <TextLink href='https://twitter.com/AsyncAPISpec' className='mt-4 inline-flex items-center' target='_blank'>
```
is there any other alternative of using inline-flex

sambhavgupta0705 commented:
@Mayaleeeee this is a design change so can you please review this

Mayaleeeee commented:
> @Mayaleeeee this is a design change so can you please review this

Is this not part of the GSoC project?

sambhavgupta0705 commented:
it is minor fix so we can merge this one
@Mayaleeeee

sambhavgupta0705 commented:
![Screenshot_2025-04-06-15-27-53-95_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/e7c0eaf5-7452-4bad-8a87-5987f64fbeb9)


This is also breaking in your changes
Kindly fix it and see for other changes

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
removed inline

PankajKumardev committed:
fix: correct typo in Twitter link class name

PankajKumardev committed:
fix: adjust width of banner container for better responsiveness

PankajKumardev commented:
> it is minor fix so we can merge this one @Mayaleeeee

Hey bro,
Updated AnnouncementHero as you mentioned , I hadn't changed it before, but fixed it now. Also removed the inline styles from the latest News section, just like you suggested.
If anything else needs to be done, just let me know.

After the changes in AnnouncementHero:

![WhatsApp Image 2025-04-07 at 11 41 21_a0e756dc](https://github.com/user-attachments/assets/02a1a3ca-24f2-425b-9217-2fe4a450ca26)

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev commented:
Just wanted to follow up on the PR I updated as per your suggestions. Let me know if there’s anything else you’d like me to address or if it's good to go. Appreciate your time and feedback!

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

PankajKumardev committed:
Merge branch 'master' into master

akshatnema committed:
Merge branch 'master' into master
